src/pykeen/models/base.py

```diff
@@ -183,7 +183,12 @@ def score_hrt(self, hrt_batch: torch.LongTensor, *, mode: Optional[InductiveMode
 
     @abstractmethod
     def score_t(
-        self, hr_batch: torch.LongTensor, *, slice_size: Optional[int] = None, mode: Optional[InductiveMode] = None
+        self,
+        hr_batch: torch.LongTensor,
+        *,
+        slice_size: Optional[int] = None,
+        mode: Optional[InductiveMode] = None,
+        ts: Optional[torch.LongTensor] = None,
```

Comment:
would `t_batch` be correct?

Reply:
I am not sure whether this would lead to confusion with `hr_batch`.

* `hr_batch` is always of shape `(batch_size, 2)`
* `ts` can be one of
  *  `None`, which translate to `(1, num_entities)`
  * `(k,)`, which translate to `(1, k)`
  * `(batch_size, k)`

Reply:
can it be `tails` then? not like we need to save so much space

Reply:
7f269bbb4d6025d4214cb4c6608009c51888bff8